beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -8,9 +8,9 @@ import (
 	ethpbv2 "github.com/prysmaticlabs/prysm/v3/proto/eth/v2"
 )
 
-//----------------
+// ----------------
```

Comment:
remove?

Reply:
This is from the original code.   (Only a space is added by IDE, so let's don't do it)